src/main/java/temporal/ChildWorkflowImpl.java

```diff
@@ -0,0 +1,28 @@
+package temporal;
+
+import io.temporal.workflow.Workflow;
+import java.time.Duration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import temporal.model.OrderInfo;
+
+// Child Workflow Implementation
+public class ChildWorkflowImpl implements ChildWorkflow {
+  private static final Logger logger = LoggerFactory.getLogger(ChildWorkflowImpl.class);
+
+  private boolean exit = false;
+
+  @Override
+  public void start() {
+    logger.info("Starting Workflow for ChildWorkflow");
+    Workflow.await(() -> exit);
+  }
+
+  @Override
+  public void sendNotification(OrderInfo orderInfo) {
+    logger.info(
+        "Sending notification for customer from child workflow: " + orderInfo.getCustomerId());
```

Comment:
Rather than string concatenation, it is recommended to use the string parameterization format for log statements, e.g. `logger.info("Sending notification for customer from child workflow: {}", orderInfo.getCustomerId());`

Reply:
@reselbob - please see the parameterization format in my comment above.  This is the appropriate approach when using an slf4j based logger.  You do not need to use `String.format` explicitly.  slf4j loggers will substitute values for the `{}` placeholders in the parameterized string template

Reply:
addressed